org.clojure/clojure/1.4.0/clj/clojure.core/conj/notes.md

```diff
@@ -0,0 +1,27 @@
+## Arities
+
+    [coll x] -> coll
+    [coll x & xs] -> coll
+
+## Docs
+
+Conjoins elements onto a collection. Returns a new collection with the
+xs 'added'. The 'addition' may happen at different 'places' depending
+on the concrete type.
+
+## Notes
+
+- If `coll` is nil, builds a list.
+- Lists (clj::clojure.core/list) are add to at the front
+- Vectors (clj::clojure.core/vector) are added to at the back
+- Queues are added to at the back
+- Maps may have `[k v]` pairs added and aren't generally ordered
```

Comment:
This is a hard one, `generally ordered` is a bit of a weasel phrase, but I'm not sure how else to explain the concept.